website/content/showcase.html

```diff
@@ -49,6 +49,11 @@ <h3>Written for gokrazy</h3>
 figurines for playing music stored on the device or directly from Spotify.
 </li>
 </ul>
```

Comment:
Your entry should go before this `</ul>`

Reply:
Ha, been a while since I've done HTML. Done.